src/ec/packages/ec-component-card/card.story.js

```diff
@@ -90,27 +92,27 @@ const formatCard = data => {
   const title = data.title.type
     ? {
         ...data.title,
-        type: text('card.title.type', data.title.type, optionalGroupId),
-        label: text('card.title.label', data.title.label, requiredGroupId),
+        type: text('card.title.type', data.title.type, tabLabels.optional),
+        label: text('card.title.label', data.title.label, tabLabels.required),
       }
     : {
         ...data.title,
-        label: text('card.title.label', data.title.label, requiredGroupId),
+        label: text('card.title.label', data.title.label, tabLabels.required),
       };
   // Description.
   const description = data.description
-    ? text('card.description', data.description, optionalGroupId)
+    ? text('card.description', data.description, tabLabels.optional)
     : false;
   // Meta.
   const meta = data.meta
-    ? array('card.meta (tags)', data.meta, '|', optionalGroupId)
+    ? array('card.meta', data.meta, '|', tabLabels.optional)
     : false;
 
   const formatTag = (tag, index) => {
     tag.label = text(
-      `card.tags.label (Tag ${index})`,
+      `card.tags[${index}].label`,
```

Comment:
Why making the index a string here? It doesn't seem necessary.

Reply:
To get this, no? 
![image](https://user-images.githubusercontent.com/14326885/81938111-c9e4df00-95f4-11ea-8d17-80b1b17f32ba.png)
we use the same also for infos and links

Reply:
oh, sorry, it's the whole link, it's correct!